modules/gdnative/godot/godot_color.cpp

```diff
@@ -28,34 +28,98 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 #include "godot_color.h"
+#include "core/variant.h"
 
-#include "color.h"
+#include "core/color.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-void _color_api_anchor() {
+void _color_api_anchor() {}
+
+void GDAPI godot_color_new_rgba(godot_color *r_dest, const godot_real p_r, const godot_real p_g, const godot_real p_b, const godot_real p_a) {
+
+	Color *dest = (Color *)r_dest;
+	memnew_placement(dest, Color(p_r, p_g, p_b, p_a));
```

Comment:
Same as with `Basis`, Color doesn't need any heap allocations and doesn't have a destructor and also doesn't have an `operator =`, so feel free to simplify the code here.